components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -1381,6 +1381,63 @@ paths:
             -H "Content-Type: application/json" -d @data.json
             "https://127.0.0.1:9443/api/am/publisher/v1/apis/890a4f4d-09eb-4877-a323-57f6ce2ed79b/revisions"'
 
+
+  ######################################################
+  # The "API Revisions" resource API
+  ######################################################
+  /apis/revisions:
+
+    #--------------------------------------------
+    # List available revisions of an API
+    #--------------------------------------------
+    get:
+      tags:
+        - API Revisions
+      summary: List available revisions of an API
+      description: |
+        List available revisions of an API
+      operationId: getAPIRevisionsWithAPIName
+      parameters:
+        - name: name
+          in: query
+          description: |
+            API Name
+          schema:
+            type: string
+        - name: version
+          in: query
+          description: |
+            Version of the API
+          schema:
+            type: string
+        - name: provider
+          in: query
+          description: |
+            API Provider
+          schema:
+            type: string
+        - name: query
+          in: query
+          schema:
+            type: string
+      responses:
+        200:
+          description: |
+            OK.
+            List of API revisions are returned.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/APIRevisionList'
+        404:
+          $ref: '#/components/responses/NotFound'
+      security:
+        - OAuth2Security:
+            - apim:api_publish
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+            "https://127.0.0.1:9443/api/am/publisher/v1/apis/revisions?apiId=2323&version=1.0.0&provider=adminquery=deployed:true"'
```

Comment:
Sample Code needs to add '&' before query

https://127.0.0.1:9443/api/am/publisher/v1/apis/revisions?apiId=2323&version=1.0.0&provider=admin&query=deployed:true